engine/src/main/java/org/terasology/rendering/opengl/GraphicState.java

```diff
@@ -81,9 +80,12 @@
     // eventually decided for GraphicState as it resides in the rendering.opengl
     // package anyway and rendering.opengl.OpenGLState felt cumbersome. --emanuele3d
 
+    // FIXME: for making nodes work without breaking anything
+    public Buffers buffers = new Buffers();
```

Comment:
If you move this line and the comment back where it used to be github will do a better job at doing the diff.